docs/output-logstash.asciidoc

```diff
@@ -270,9 +250,27 @@ The provided identity will be used if the downstream input enables <<plugins-{ty
 ** `none`: performs no validation of the presented certificate
```

Comment:
scrolling up, we also need:

~~~ diff
@@ -268,1 +248,1 @@
-* There is only one currently-supported mode:
+* The supported modes are:
~~~